osu.Game/Screens/Play/BreakOverlay.cs

```diff
@@ -109,6 +117,33 @@ protected override void LoadComplete()
             initializeBreaks();
         }
 
+        protected override void Update()
+        {
+            base.Update();
+
+            updateBreakTimeBindable();
+        }
+
+        private void updateBreakTimeBindable()
+        {
+            if (breaks == null)
+                return;
+
+            for (int i = 0; i < breaks.Count; i++)
```

Comment:
you haven't changed anything.

you should not be iterating. store an index and use the index to proceed forward or backwards or not.

Reply:
Resolved, using your proposal (storing a global index and using it to proceed), sorry for my misunderstanding.